Packs/FeedAlienVault/Integrations/FeedAlienVaultOTXTaxii/FeedAlienVaultOTXTaxii.py

```diff
@@ -0,0 +1,667 @@
+                        f"configuration on Alien Vault")
+
+    if len(passed_collections) == 0:
+        raise Exception("Unable to poll from any collection - please check the configuration on Alien Vault")
+
+    return 'ok', {}, {}
+
+
+def get_indicators_command(client: Client, args: Dict):
+    """Runs fetch indicators and return the indicators.
+
+    Args:
+        client(Client): The AlienVault OTX client.
+        args(dict): The command arguments
+
+    Returns:
+        str,dict,dict. The human readable, and rawJSON from the command - no context created.
+    """
+    limit = int(args.get('limit', 50))
+    indicator_list = fetch_indicators_command(client, limit)
+
+    human_readable = tableToMarkdown("Indicators from AlienVault OTX TAXII:", indicator_list,
+                                     headers=['value', 'type'], removeNull=True)
+
+    return human_readable, {}, indicator_list
+
+
+def parse_indicators(sub_indicator_list, full_indicator_list):
+    """Gets a decoded indicator list and returns a parsed version of the indicator with accordance with Demisto's
+    Feed indicator standards.
+
+    Args:
+        sub_indicator_list(list): A list of STIXDecoded indicators
+        full_indicator_list(list): A list of all the indicators fetched to this point - used to prevent duplications.
+
+    Returns:
+        list,list. A list of parsed indicators and an updated list of all indicators polled
+    """
+    parsed_indicator_list = []  # type: List
+    for indicator in sub_indicator_list:
+        if indicator['indicator'] in full_indicator_list:
+            continue
+
+        indicator['value'] = indicator['indicator']
+        del indicator['indicator']
+        temp_copy = indicator.copy()
+        indicator['rawJSON'] = temp_copy
+        indicator['fields'] = {
+            "description": indicator["stix_package_short_description"]
+        }
+        parsed_indicator_list.append(indicator)
+        full_indicator_list.append(indicator['value'])
+
+    return parsed_indicator_list, full_indicator_list
+
+
+def fetch_indicators_command(client: Client, limit=None):
+    """Fetch indicators from AlienVault OTX.
+
+    Args:
+        client(Client): The AlienVault OTX client.
+        limit(any): How many XML elements to parse, None if all should be parsed.
+
+    Returns:
+        list. A list of indicators.
+    """
+    indicator_list = []  # type:List
+    for collection in client.collections:
+        try:
+            taxii_iter = client.build_iterator(collection)
+
+        except Exception as e:
+            if not client.all_collections:
+                raise Exception(e)
+
+            else:
+                continue
+
+        index = 0
+        only_indicator_list = []  # type:List
+        for raw_response in taxii_iter:
+            _, res = client.decode_indicators(raw_response.content)
+            parsed_list, only_indicator_list = parse_indicators(res, only_indicator_list)
```

Comment:
you are not using only_indicator_list anywhere...
And why are you sending an empty list to the parse_indicators function?
I think that this line should be 
```suggestion
            parsed_list, _ = parse_indicators(res, indicator_list)
```
Please check that